xplique/attributions/image_object_detector_explainer.py

```diff
@@ -0,0 +1,214 @@
+"""
+Module related to Object detector method
+"""
+from typing import Iterable, Tuple, Union, Optional
+import abc
+import tensorflow as tf
+import numpy as np
+from xplique.attributions.base import BlackBoxExplainer
+from xplique.commons.tf_operations import inference_batching
+
+
+class IIouCalculator:

```

Comment:
Is it on purpose that there is two capital I . Does it make sense to make the following change:
"IIouCalculator -> IouCalculator" ?

Reply:
Done.